types/aframe/tests/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Yeah, if DT doesn't include tests in it's CI then a separate repo is almost as good as including code that may not be run by an engineer editing these types. I'll spin off this material in a separate repo and add comments to `index.d.ts`